Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.csproj

```diff
@@ -68,13 +68,16 @@
   <Target Name="SetUpGlobalESLintConfiguration" BeforeTargets="BeforeResolveReferences">
     <PropertyGroup>
       <NodeJsExtensionsGlobalESLintConfigurationDirectory
-        Condition="'$(NodeJsExtensionsGlobalESLintConfigurationDirectory)' != ''
+        Condition="'$(NodeJsExtensionsGlobalESLintConfigurationDirectory)' == ''
+      ">$(ProjectDir)</NodeJsExtensionsGlobalESLintConfigurationDirectory>
+      <NodeJsExtensionsGlobalESLintConfigurationDirectory
+        Condition="'$(NodeJsExtensionsGlobalESLintConfigurationDirectory)' != 'false'
       ">$(NodeJsExtensionsGlobalESLintConfigurationDirectory.TrimEnd('/\'))</NodeJsExtensionsGlobalESLintConfigurationDirectory>
```

Comment:
On second thought @0liver, why is TrimEnd necessary here? I think if you have an extra slash in the path that won't actually hurt. I think this should be removed